test/SlowTests/Issues/RavenDB-17513.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Client;
+using Raven.Client.Documents.Smuggler;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_17513 : RavenTestBase
+    {
+        public RavenDB_17513(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Theory]
+        [InlineData(1024)]
+        [InlineData(2 * 1024 + 100)]
+        [InlineData(10_000)]
+        public async Task Can_Import_Subscriptions(int count)
+        {
+            using (var store1 = GetDocumentStore())
+            using (var store2 = GetDocumentStore())
+            {
+                for (var i = 0; i < count; i++)
+                {
+                    await store1.Subscriptions.CreateAsync<Query.Order>(
+                        order => order.Lines.Any(x => x.Discount > 10)
+                    );
+                }
+
+                var operation = await store1.Smuggler.ExportAsync(new DatabaseSmugglerExportOptions
```

Comment:
I'm actually thinking what will happen to other tests when this command is executed? This test is not using a local server. @karmeli87 maybe the source of those random timeouts in the tests is related?

Reply:
Hmm it is a possibility... basically if we are queuing up faster than we can process the timeout exception will be thrown..
We have the `RachisLogHistory` where we mark the time when we appended the command and we update it later when we do commit, maybe for testing we can print out the command if `commit - append > 1s`